domain/src/unit-test/java/uk/gov/moj/sdt/domain/IndividualRequestTest.java

```diff
@@ -31,149 +31,146 @@
 
 package uk.gov.moj.sdt.domain;
 
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
-
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 import uk.gov.moj.sdt.domain.api.IBulkSubmission;
 import uk.gov.moj.sdt.domain.api.IErrorLog;
 import uk.gov.moj.sdt.domain.api.IIndividualRequest;
 import uk.gov.moj.sdt.domain.api.IIndividualRequest.IndividualRequestStatus;
 import uk.gov.moj.sdt.utils.AbstractSdtUnitTestBase;
 
+import java.time.LocalDateTime;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
 /**
  * Unit tests for {@link IndividualRequest}.
  *
  * @author Saurabh Agarwal
  */
-public class IndividualRequestTest extends AbstractSdtUnitTestBase {
+class IndividualRequestTest extends AbstractSdtUnitTestBase {
     /**
      * Test subject.
      */
     private IIndividualRequest individualRequest;
 
+    private static final String STATUS_IS_INCORRECT = "Status is incorrect";
+    private static final String FORWARDING_ATTEMPT_COUNT_IS_INCORRECT = "Forwarding attempt count is incorrect";
+    private static final String UPDATED_DATE_SHOULD_BE_POPULATED = "Updated date should be populated";
+
     /**
      * Set up test data.
      */
-    @Before
+    @BeforeEach
+    @Override
     public void setUp() {
         individualRequest = new IndividualRequest();
     }
 
+    private void assertsCommon(IndividualRequestStatus expectedStatus, Integer expectedForwardingAttempts,
+                               LocalDateTime updatedDate) {

```

Comment:
Looks like the updatedDate parameter can be removed.  When the assertCommon() method is called the value of this parameter is only ever set to individualRequest.getUpdatedDate().  The method already uses individualRequest.getRequestStatus() and getForwardingAttempts(), so it might as well use getUpdatedDate() as well.